### PR TITLE
Portfolio - mobile menu popup

### PR DESCRIPTION
**Pull request by chibunduonyeje1**
### The project is built with following

- Add resource files such as images, CSS file and JavaScript to Assets folder
- Created a div to show the menu icons
- used JavaScript to Add the effect to display and not to display
- Fixed linters errors for html, CSS and JavaScript

### CODE REVIEW

File: assets/js/window-pop.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  card.classList.add('card');
+  workSection.append(card);
+  card.innerHTML = `<img src="${projectData[i].image}" alt="">
+    <div class="cardComponent">
+        <h2>${projectData[i].title}</h2>
+        <p>${projectData[i].description}</p>
+        <ul class="stacktwo">${techData}</ul>
+        <button type="button" class="worktwo-button">See Project</button>
+    </div>`;
+}
```
Comment (Olamarx) [inline assets/js/window-pop.js]: - [ ] It is best practice to have your code organized by using functions. I would recommend you add all these code in a function and then call the function. Remember to use the es6 arrow function and not the function keyword to show you are updated.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @chibunduonyeje1, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Mobile Menu functionalities ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @chibunduonyeje1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no changes as requested, kindly make the changes as they are required.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by OyePriscilla — APPROVED** [review]
Hi @chibunduonyeje1 👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 
GitHub flow observed 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**